changes/7fc586cfa9ba67b1e7bb1344899daed4.yaml

```diff
@@ -0,0 +1,7 @@
+---
+desc: Changed the ``synapse.tests.utils.SynTest`` class to inherit from ``unittest.IsolatedAsyncioTestCase``.
```

Comment:
This feels a bit engineering specific and prone to users getting confused. Is this an internal adjustment that doesn't warrant a "feature" log entry?

Reply:
It is an engineering specific change that could affect any downstream Synapse package user that is using our `synapse.tests.utils.SynTest` class. This would include folks that have test harnesses for advanced power-ups or rapid powerups.